### PR TITLE
Introduce stats on the front page

### PR DESCRIPTION
**Pull request by shtukas**
This commit contains the first part of the two parts change introducing stats on the front end.

### CODE REVIEW

**Comment by shtukas** [comment]
screen shot 2016-12-07 at 20 20 38

**Comment by shtukas** [comment]
Note:

`userStatsBiggestDayDate`, `userStatsBiggestDayCount`, `userStatsRanking` are in TODO state. They feed the below HTML fragment 

```
<!--
    <span class=""><strong>Biggest day:</strong> </span><span class="">@{userStats(PStatsBiggestDayDate)}, @{userStatsPStatsBiggestDayCount)} recipes</span><br />
    <span class=""><strong>How you compare: </strong> </span><span class="">@{userStats(PStatsRanking)}</span>
-->
```

which is going to be uncommented when the functions are done.

File: ui/app/com/gu/recipeasy/views/app.scala.html [hunk 1]
```diff
@@ -8,18 +13,35 @@
...
-                Stage one (structuring) completion @{progressBarPercentage}%
+                <div class="text-xs-center" id="curation-caption">
+                    Stage one (structuring) completion @{progressBarPercentage}%
+                </div>
+                <progress class="progress" value="@{progressBarPercentage}" max="100" aria-describedby="curation-caption"></progress>
+            </div>
+        </div>
+        <div class="container" style="padding:30px 0 20px 0;background-color: rgba(255, 255, 255, 0.6);">
+            <div class="col-md-6">
+                <h1>Hi @{userFirstname}, your stats</h1>
```
Comment (alastairjardine) [inline ui/app/com/gu/recipeasy/views/app.scala.html]: What about:
`<username>'s stats`?

Comment (alastairjardine) [inline ui/app/com/gu/recipeasy/views/app.scala.html]: it is a bit tighter

File: ui/app/com/gu/recipeasy/views/app.scala.html [hunk 2]
```diff
@@ -8,18 +13,35 @@
...
+                <span class=""><strong>Contributions:</strong> </span><span class="">@{userStats(PStatsCurationCount)} curated, @{userStats(PStatsVerificationCount)} verified, @{userStats(PStatsFinalisationCount)} finalised, @{yourParticipation(generalStats, userStats)}</span><br />
```
Comment (alastairjardine) [inline ui/app/com/gu/recipeasy/views/app.scala.html]: Let's split out percentage contributed on to a new line
`Percentage contribution: 0.04%`

**Comment by shtukas** [comment]
@alastairjardine I pushed the changes you requested

screen shot 2016-12-08 at 09 45 44

**Comment by alastairjardine** [comment]
👍 

Make it so!
make it so

**Comment by shtukas** [comment]
Awesome!

**Pull request merged**